common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -74,7 +74,7 @@
  * {@code sun.misc.Unsafe} object.
  * <p>
  * You can disable the use of {@code sun.misc.Unsafe} if you specify
- * the system property <strong>io.netty.noUnsafe</strong>.
+ * the system property <strong>siren.io.netty.noUnsafe</strong>.
  */
 public final class PlatformDependent {
```

Comment:
It seems not all changes for this class was applied, can you please check, see https://github.com/sirensolutions/netty/pull/34

Reply:
Actually, if we take a look at the commits, the changes were already applied in this PR - 8221353d4bd974d552391d0ab88bbf90ee97dbc0 and 20c16d8df48d9010f81b853512d6e106376f2534. I think the changes won't show up in the overall changes as the commits from https://github.com/sirensolutions/netty/pull/34 depicts the memory settings removal.

Reply:
I am not sure I understand your commit with regards to the `PlatformDependent` class. Can you please compare the version on [branch-siren-4.1.82.x](https://github.com/sirensolutions/netty/tree/branch-siren-4.1.82.x) to your pr? That is the branch to be used as reference

Reply:
Maybe you should check if the pr https://github.com/sirensolutions/netty/pull/34 reverted the changes we made on the fork to the original behavior. You can compare the changes to the version of the class in [branch-siren-4.1.104.x](https://github.com/sirensolutions/netty/tree/branch-siren-4.1.104.x)